### PR TITLE
Fix documentation errors.

### PR DESCRIPTION
**Pull request by damien-carcel**
This PR
- [x] adds a new section for the queue implementation, but only in the Docker part; the manual install procedure will be updated later,
- [x] fix several mistakes,
- [x] add info about Elasticsearch `vm.max_map_count`,
- [x] move the index so there is only two entries for install: "Docker" and "Manual".

### CODE REVIEW

File: install_pim/docker/installation_docker.rst [hunk 1]
```diff
@@ -38,9 +38,28 @@ Otherwise ``docker-compose`` will create it, but only with root accesses. Then f
 Run and stop the containers
 ---------------------------
 
-**All "docker-compose" commands are to be ran from the folder containing the compose file.**
+.. note::
+
+   All "docker-compose" commands are to be ran from the folder containing the compose file.
```
Comment (tamarasaurus) [inline install_pim/docker/installation_docker.rst]: to be run

File: install_pim/docker/installation_docker.rst [hunk 2]
```diff
@@ -38,9 +38,28 @@ Otherwise ``docker-compose`` will create it, but only with root accesses. Then f
...
+
+.. warning::
+
+   To run the Elasticsearch container, you will probably need to `increase the MAX_MAP_COUNT Linux kernel setting <https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html#docker-cli-run-prod-mode>`_.
```
Comment (tamarasaurus) [inline install_pim/docker/installation_docker.rst]: 'probably' sounds a bit vague for documentation... you could say 'might need to'

File: install_pim/docker/installation_docker.rst [hunk 3]
```diff
@@ -38,9 +38,28 @@ Otherwise ``docker-compose`` will create it, but only with root accesses. Then f
...
+   Proceed as follow (first command will affect your current session, second one every boot of your machine):
```
Comment (tamarasaurus) [inline install_pim/docker/installation_docker.rst]: as follows

File: install_pim/docker/installation_docker.rst [hunk 4]
```diff
@@ -157,6 +176,37 @@ The version in ``akeneo/pim-community-standard`` or ``akeneo/pim-enterprise-stan
 
    $ docker-compose run --rm node yarn run webpack
 
+
+Run imports and exports
+***********************
+
+Akeneo 2.x implement a queue for the jobs, as a PHP daemon. This daemon is a Symfony command, that can only executes one job at a time. It does not consume any other job until the job is finished.
```
Comment (tamarasaurus) [inline install_pim/docker/installation_docker.rst]: implements

Comment (damien-carcel) [reply install_pim/docker/installation_docker.rst]: Yep, just saw this one ><

File: install_pim/docker/installation_docker.rst [hunk 5]
```diff
@@ -157,6 +176,37 @@ The version in ``akeneo/pim-community-standard`` or ``akeneo/pim-enterprise-stan
 
    $ docker-compose run --rm node yarn run webpack
 
+
+Run imports and exports
+***********************
+
+Akeneo 2.x implements a queue for the jobs, as a PHP daemon. This daemon is a Symfony command, that can only executes one job at a time. It does not consume any other job until the job is finished.
```
Comment (rybus) [inline install_pim/docker/installation_docker.rst]: execute

**Pull request merged**